Subject: add MODULE_DEVICE_TABLE to ehea and ehca driver
From: olh@suse.de
References: bnc#394602 - LTC44938
Patch-mainline: no

modinfo ehea
...
alias:          of:NlheaT*CIBM,lhea*
...

cat  /sys/bus/ibmebus/devices/23c00100.lhea/modalias 
of:NlheaT<NULL>CIBM,lhea


---
 drivers/infiniband/hw/ehca/ehca_main.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/infiniband/hw/ehca/ehca_main.c
+++ b/drivers/infiniband/hw/ehca/ehca_main.c
@@ -923,6 +923,7 @@ static struct of_device_id ehca_device_t
 	},
 	{},
 };
+MODULE_DEVICE_TABLE(of, ehca_device_table);
 
 static struct of_platform_driver ehca_driver = {
 	.name        = "ehca",
